spec/controllers/verify/usps_controller_spec.rb

```diff
@@ -25,6 +26,37 @@
 
       expect(response).to be_ok
     end
+
+    it 'redirects if the user has sent too much mail' do
+      usps_mail_service = Idv::UspsMail.new(user)
+      allow(usps_mail_service).to receive(:mail_spammed?).and_return(true)
+      allow(subject.idv_session).to receive(:address_mechanism_chosen?).
+        and_return(true)
+      get :index
+
+      expect(response).to redirect_to verify_review_path
```

Comment:
@zachmargolis Here. at runtime all the requisites are right for a redirect, but it still is returning an `ok` for me